specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -2639,37 +2879,35 @@
             "$ref": "#/parameters/api-version"
           },
           {
-            "name": "$top",
-            "description": "The number of items to be included in the result.",
-            "in": "query",
-            "required": false,
-            "type": "integer",
-            "format": "int32"
+            "name": "parameters",
+            "description": "The callback URL parameters.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GetCallbackUrlParameters"
+            }
           }
         ],
         "responses": {
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/IntegrationAccountCertificateListResult"
+              "$ref": "#/definitions/CallbackUrl"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/certificates/{certificateName}": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/listKeyVaultKeys": {
+      "post": {
         "tags": [
-          "IntegrationAccountCertificates"
+          "IntegrationAccounts"
         ],
-        "operationId": "Certificates_Get",
-        "description": "Gets an integration account certificate.",
+        "operationId": "IntegrationAccounts_ListKeyVaultKeys",
+        "description": "Gets the integration account's Key Vault keys.",
```

Comment:
As a list operation this should probably also be pageable.

Reply:
Added